modules/tftp/server.rb

```diff
@@ -110,7 +110,7 @@ def pxe_default
       [pxeconfig_dir]
     end
     def pxeconfig_file mac
-      ["#{pxeconfig_dir}/"+mac.gsub(/:/,"").upcase]
+      ["#{pxeconfig_dir}/"+mac.gsub(/:/,"").upcase+".cfg"]
```

Comment:
Can you actually return the original and then the new (copy) with added extension for backward compatibility? Proxy will render two files with the same content actually.

Reply:
Sure! Will change it tonight 😊 thanks for the review!